src/executables/ui/status.ts

```diff
@@ -0,0 +1,59 @@
+import * as vscode from 'vscode';
+
+import { isVirtual } from '../service/class';
+import { RExecutableService } from '../service';
+
+enum BinText {
+    name = 'R Language Indicator',
+    missing = '$(warning) Select executable'
+}
+
+export class ExecutableStatusItem implements vscode.Disposable  {
+    private readonly service: RExecutableService;
+    private languageStatusItem: vscode.LanguageStatusItem;
```

Comment:
vscode 1.65.0 seems to be the lowest version that supports this api

Reply:
Right, I believe it's definitely a newer addition. I'll look into creating an adapter class of some sort for older versions

Reply:
We can just change the minimum version of `@types/vscode` and `"engines": {"vscode": ...}` to 1.65, I'd suggest.